### PR TITLE
Read in COVID data and expose to API

### PR DESCRIPTION
**Pull request by camenpihor**
## Changes

1. Move COVID data
2. Add COVID data to MANIFEST.in so that its included in packaging
3. Remove unused data from COVID data to decrease package size
4. Add method to read in COVID confirmed case data
5. Test method to read in COVID data
6. Test resource existence
6. Decide to not use daily reports since the formatting between files changes a lot... We can get the same thing, I think, from the time series data

### CODE REVIEW

**Comment by mjhoy** [comment]
or is the intention not to use a submodule

File: backend/api/app.py [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+    """Ping test."""
     return "Success"
 
 
 @app.route("/covid/confirmed/new/<date>")
 def get_new_confirmed(date):
     """Get new confirmed COVID-19 cases on date."""
-    return f"new on {date}"
+    print(date)
+    return read_confirmed_time_series().to_json()
```
Comment (mjhoy) [inline backend/api/app.py:19]: could we read in the CSV at run time and just keep it as a global?

**Comment by camenpihor** [comment]
I would pair on the submodule stuff, I don't understand it myself. I
thought that I had done it correctly

**Comment by camenpihor** [comment]
Hmm, I think I need to hear more about what you mean, so my answer is
maybe! Code is definitely nowhere near optimized for what it should look
like given calls that it would receive. Waiting to see more about what
kinds of things the frontend wants

**Comment by mjhoy** [comment]
> Waiting to see more about what
kinds of things the frontend wants

yeah, probably not worth thinking about too much right now.

let's pair on the submodule!

**Pull request merged**